docs/04-how-we-work/holacracy.md

```diff
@@ -0,0 +1,111 @@
+# Holacracy
```

Comment:
@nerdstein I feel this page should be "pure". No need to talk about CivicActions here, just describe what holocracy is. In another page we can talk about how we do it, and how what we do connects to holocracy. The same thing with agile guilds. We can have a pure page about what the agile guild concept is, and in other pages regarding what we do we can link to the pure pages.